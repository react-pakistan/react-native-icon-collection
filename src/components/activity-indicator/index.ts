export {
  ActivityIndicator,
  ActivityIndicatorSize,
} from './activity-indicator';
