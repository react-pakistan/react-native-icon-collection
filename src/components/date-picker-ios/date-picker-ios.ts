import styled from 'styled-components/native';

export const DatePickerIOS = styled.DatePickerIOS``;
