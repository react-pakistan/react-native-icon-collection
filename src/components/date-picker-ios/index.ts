export { DatePickerIOS } from './date-picker-ios';
