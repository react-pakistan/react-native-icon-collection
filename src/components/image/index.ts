export {
  IImageProps,
  Image,
  ResizeMethod,
  ResizeMode,
} from './image';
