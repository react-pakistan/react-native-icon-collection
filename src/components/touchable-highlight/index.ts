export { TouchableHighlight } from './touchable-highlight';
