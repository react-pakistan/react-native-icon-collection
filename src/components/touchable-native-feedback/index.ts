export { TouchableNativeFeedback } from './touchable-native-feedback';
