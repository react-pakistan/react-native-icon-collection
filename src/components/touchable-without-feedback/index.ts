export { TouchableWithoutFeedback } from './touchable-without-feedback';
