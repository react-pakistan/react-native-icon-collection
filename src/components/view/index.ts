export { View, IView } from './view';
