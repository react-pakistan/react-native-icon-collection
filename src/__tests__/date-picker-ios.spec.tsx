import React from 'react';
import renderer from 'react-test-renderer';
import { DatePickerIOS } from '../components/date-picker-ios';
import {
  datePickerIOSProps,
} from '../../__stories__/story/date-picker-ios.story';

describe('Testing DatePickerIOS component', () => {
  it('matches DatePickerIOS snapshot', () => {
    const wrapper = renderer.create(
      <DatePickerIOS {...datePickerIOSProps} />,
    );
    expect(wrapper.toJSON()).toMatchSnapshot();
  });
});
