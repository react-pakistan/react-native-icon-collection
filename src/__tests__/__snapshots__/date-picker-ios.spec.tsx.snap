// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing DatePickerIOS component matches DatePickerIOS snapshot 1`] = `
<View
  style={
    Array [
      Object {},
    ]
  }
>
  <RCTDatePicker
    date={532465200000}
    locale="en"
    maximumDate={1604862000000}
    minimumDate={1573239600000}
    onChange={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "height": 216,
      }
    }
    timeZoneOffsetInMinutes={-300}
  />
</View>
`;
